pkg/client/client.go

```diff
@@ -208,6 +209,7 @@ func newControllerCliOptions() controllercli.Options {
 	mapper.Add(v1alpha1.SchemeGroupVersion.WithKind(meta.KindKogitoJobsService.Name), &restScope{name: apimeta.RESTScopeNameNamespace})
 	mapper.Add(v1alpha1.SchemeGroupVersion.WithKind(meta.KindKogitoInfra.Name), &restScope{name: apimeta.RESTScopeNameNamespace})
 	mapper.Add(v1alpha1.SchemeGroupVersion.WithKind(meta.KindKogitoMgmtConsole.Name), &restScope{name: apimeta.RESTScopeNameNamespace})
+	mapper.Add(v1alpha1.SchemeGroupVersion.WithKind(meta.KindKogitoRuntime.Name), &restScope{name: apimeta.RESTScopeNameNamespace})
```

Comment:
This change should be done as part of the original Zanini's PR. I've already commented it in there.